user/love/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "love"
+pkgver = "11.5"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
```

Comment:
this needs a comment why

Reply:
so, i looked at Alpine's APKBUILD file and tried to build it and it's from there. I built and tested it again without it and it works. sorry.